exporter/datadogexporter/examples/collector.yaml

```diff
@@ -372,7 +372,7 @@ exporters:
       ##
       ## To attach tags to telemetry signals regardless of the host, use a processor instead.
```

Comment:
And it would be very good to refer to something (link) with what is meant as "processor". I would like to add team tag on signals and I cannot use hosttags because bunch of different applications developed by different teams are sending signals to the same collector. With hosttags they would all get the same team tag.

Reply:
What is meant by processor are the processors mentioned here: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor (e.g. `resourceprocessor`, `attributesprocessor`). I think linking to the processor link above makes sense here 👍

Reply:
Should I send another patch?

Reply:
Sounds good to me @utezduyar 👍

Reply:
Can you please help me with some guidelines so I can update this PR with the processor link. Below is my confusion.

If one uses resource or attribute processor, I believe the tags will show up in DD as span tags. How can someone send platform tags (for example a team tag) through these processors? Is there a magic syntax or another way that I am missing? For example I tried sending attribute "_dd.tag.team=foo" but that didn't work.

Reply:
#1378844

Reply:
The support case I opened became irrelevant for this discussion.